src/tree/GroupTreeItemBase.ts

```diff
@@ -58,13 +59,11 @@ export class GroupTreeItemBase extends AzExtParentTreeItem {
     }
 
     public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
-        if (clearCache) {
-            // this.treeMap = {};
-        }
-
         for (const ti of Object.values(this.treeMap)) {
             if (ti instanceof ResolvableTreeItemBase) {
-                void ti.resolve(clearCache, context);
+                if (!(ti.getResolver() instanceof WrapperResolver)) {
+                    void ti.resolve(clearCache, context);
```

Comment:
If we're resolving in the tree framework, we shouldn't need to resolve here, regardless of type.